yarnitor/static/js/main.js

```diff
@@ -89,43 +92,56 @@ $(document).ready(function() {
                 data: "name",
                 title: "Name",
                 render: function(data, type, row, meta) {
-                    if(data.length > MAX_NAME_LENGTH) {
-                        data = data.substr(0, MAX_NAME_LENGTH) + '...';
+                    if(type === 'display') {
```

Comment:
what are the other values for "type"

Reply:
https://datatables.net/reference/option/columns.render in the function render box and unfortunately I can't link any deeper it seems. :/